pyflakes/test/test_api.py

```diff
@@ -55,26 +55,50 @@ def __init__(self, lineno, col_offset=0):
 
 class SysStreamCapturing(object):
 
-    """Replaces sys.stdin, sys.stdout and sys.stderr with StringIO objects."""
+    """
```

Comment:
Fair. I checked the example in the document and skimmed over the portion where it said it can be written like this or on the same line.